spring-cloud-dataflow-docs/src/main/asciidoc/index.adoc

```diff
@@ -24,13 +24,13 @@ Sabby Anandan; Marius Bogoevici; Eric Bottard; Mark Fisher; Ilayaperumal Gopinat
 
 :dataflow-asciidoc: https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/spring-cloud-dataflow-docs/src/main/asciidoc
 
-:docker-http-source-rabbit-version: 1.3.1.RELEASE
-:docker-time-source-rabbit-version: 1.3.1.RELEASE
-:docker-log-sink-rabbit-version: 1.3.1.RELEASE
-:docker-log-sink-kafka-version: 1.3.1.RELEASE
-:docker-http-source-kafka-version: 1.3.1.RELEASE
-:docker-time-source-kafka-version: 1.3.1.RELEASE
-:docker-timestamp-task-version: 1.3.1.RELEASE
+:docker-http-source-rabbit-version: 2.1.0.RELEASE
+:docker-time-source-rabbit-version: 2.1.0.RELEASE
+:docker-log-sink-rabbit-version: 2.1.0.RELEASE
+:docker-log-sink-kafka-version: 2.1.0.RELEASE
+:docker-http-source-kafka-version: 2.1.0.RELEASE
+:docker-time-source-kafka-version: 2.1.0.RELEASE
+:docker-timestamp-task-version: 2.1.0.RELEASE
```

Comment:
We can't switch `timestamp` to 2.1 GA yet; we are waiting on SCDF 2.0 to release the Task train, so it'd be good to revert this to 1.3.1 or 2.0.1.